.github/workflows/check-out-of-tree-build.yml

```
@@ -0,0 +1,72 @@
+# This workflow is intended to check if PR doesn't introduce any regressions
+# into out-of-tree build in various configurations.
+#
+# Documentation for GitHub Actions:
+# [workflow-syntax]: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+# [context-and-expression-syntax]: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions
+name: Out-of-tree build & tests
+
+on:
+  push:
+    branches:
+      - master
+      - llvm_release_*
+    paths-ignore: # no need to check build for documentation changes
+      - 'docs/**'
+      - '**.md'
+  pull_request:
+    branches:
+      - master
+      - llvm_release_*
+    paths-ignore: # no need to check build for documentation changes
+      - 'docs/**'
+      - '**.md'
+  schedule:
+    - cron: 0 0 * * *
+
+env:
+  LLVM_VERSION: 12
+
+jobs:
+  build_and_test:
+    name: Linux
+    strategy:
+      matrix:
+        build_type: [Release, Debug]
+      fail-fast: false
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Install dependencies
+        run: |
+          curl -L "https://apt.llvm.org/llvm-snapshot.gpg.key" | sudo apt-key add -
+          curl -L "https://packages.lunarg.com/lunarg-signing-key-pub.asc" | sudo apt-key add -
+          echo "deb https://apt.llvm.org/bionic/ llvm-toolchain-bionic main" | sudo tee -a /etc/apt/sources.list
+          echo "deb https://packages.lunarg.com/vulkan bionic main" | sudo tee -a /etc/apt/sources.list
+          sudo apt-get update
+          # Linux systems in GitHub Actions already has pre-installed clang is
```

The same comment as @svenvh left for in-tree build config
```suggestion
          # Linux systems in GitHub Actions already have a pre-installed clang that is
```

---

Fixed in b360b08f00ea63d7a488c22037fe18ff7b5ca446